pkg/config/config.go

```diff
@@ -82,6 +82,9 @@ type ClientConfig struct {
 	RemoteUsers []string `yaml:"remote_users"`
 	// bless calls these bastion ips - your source ip. 0.0.0.0/0 is all
 	BastionIPS []string `yaml:"bastion_ips"`
+	// ask bless to validate existing certs against the remote users
+	// the default is true.
+	ValidatePrincipals *bool `yaml:"validate_principals"`
```

Comment:
nit: To simplify this logic a bit and make the 0 value useful we can instead change this to `SkipPrincipalValidation` (which defaults false). 
Then you can change the bottom logic to something like

`isFresh := isFresh && (SkipPrincipalValidation || validatePrincipals())` or something like that